src/sentry/static/sentry/app/views/dataExport/dataDownload.tsx

```diff
@@ -174,9 +175,11 @@ class DataDownload extends AsyncView<Props, State> {
               {err.status} - {err.statusText}
             </h3>
           </Header>
-          <Body>
-            <p>{err.responseJSON.detail}</p>
-          </Body>
+          {errDetail && (
+            <Body>
+              <p>{errDetail}</p>
+            </Body>
+          )}
```

Comment:
This is Point 3. If there isn't an error message, don't show this body section (as is the case in 404s)